CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+| bug fix |  **datepicker:** label state not being updated in some cases ([#19761](https://github.com/angular/components/issues/19761)) ([0a22e0f](https://github.com/angular/components/commit/0a22e0f)) |
+| bug fix |  **datepicker:** not marking as dirty when invalid value is typed in ([#19730](https://github.com/angular/components/issues/19730)) ([590fdd2](https://github.com/angular/components/commit/590fdd2)), closes [#19726](https://github.com/angular/components/issues/19726) |
+| bug fix |  **datepicker:** range input separator not hidden in high contrast mode ([#19706](https://github.com/angular/components/issues/19706)) ([2894a71](https://github.com/angular/components/commit/2894a71)) |
+| bug fix |  **drag-drop:** error when cloning 0x0 canvas ([#19707](https://github.com/angular/components/issues/19707)) ([92bc813](https://github.com/angular/components/commit/92bc813)) |
+| bug fix |  **drag-drop:** placeholder element not available in started event ([#19493](https://github.com/angular/components/issues/19493)) ([94d3fda](https://github.com/angular/components/commit/94d3fda)), closes [#19457](https://github.com/angular/components/issues/19457) |
+| bug fix |  **drag-drop:** sometimes incorrectly swapping items at the ends of the list ([#19633](https://github.com/angular/components/issues/19633)) ([63fe5ed](https://github.com/angular/components/commit/63fe5ed)), closes [#19249](https://github.com/angular/components/issues/19249) |
+| bug fix |  **drag-drop:** update free drag position on scroll ([#19648](https://github.com/angular/components/issues/19648)) ([ebbe8d7](https://github.com/angular/components/commit/ebbe8d7)) |
+| bug fix |  **focus indicators:** Improve styles that determine when focus indicators should render on selection controls. ([#19546](https://github.com/angular/components/issues/19546)) ([c85920d](https://github.com/angular/components/commit/c85920d)) |
+| bug fix |  **form-field:** error in older versions of edge ([#19698](https://github.com/angular/components/issues/19698)) ([539ebf6](https://github.com/angular/components/commit/539ebf6)), closes [#17810](https://github.com/angular/components/issues/17810) |
+| bug fix |  **input:** incorrectly handling nested themes ([#19637](https://github.com/angular/components/issues/19637)) ([4a8ce63](https://github.com/angular/components/commit/4a8ce63)) |
+| bug fix |  **sidenav:** restore focus if drawer is closed through backdrop click ([#19534](https://github.com/angular/components/issues/19534)) ([c7eaec9](https://github.com/angular/components/commit/c7eaec9)), closes [#17749](https://github.com/angular/components/issues/17749) |
+
+### material-experimental
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **mdc-form-field:** update input alignment to match spec ([27f5271](https://github.com/angular/components/commit/27f5271)) |
+
+### progress-spinner
```

Comment:
I think this belongs under the material section

Reply:
Fixed